ui/src/components/timetables/import/FileImportDragAndDrop.tsx

```diff
@@ -16,6 +16,10 @@ interface Props {
   setFileList: (fileList: File[]) => void;
 }
 
+const isCorrectFormatFile = (file: File) =>
+  // exp files have empty file type, just use filename extension for checking instead
+  file.type === 'text/csv' || file.name.split('.')[1] === 'exp';
```

Comment:
you could use `file.name.split('.').pop()` to actually get the extension and this would work even if the file would have dots in it's name. Very unlikely that we would have this issue, but the more sure the better :)